gym/envs/classic_control/pendulum.py

```diff
@@ -142,8 +147,24 @@ def reset(
         options: Optional[dict] = None
     ):
         super().reset(seed=seed)
-        high = np.array([np.pi, 1])
-        self.state = self.np_random.uniform(low=-high, high=high)
+        if options is None:
+            high = np.array([np.pi, 1])
+        else:
+            x = options.pop('x', DEFAULT_X)
+            y = options.pop('y', DEFAULT_Y)
+            # We expect only numerical inputs.
+            assert option_parser.verify_number(x)
+            assert option_parser.verify_number(y)
+            # Since the same boundaries are used for all observations, we set the
+            # limits according to the most restrictive (sin/cos). Since these are
+            # the values that will be used for the `high` variable, we enforce them
+            # to be non-negative: (0., 1.).
+            x = max(0.0, min(1.0, x))
```

Comment:
The bounds here seem off. Keep in mind that this code manages the state, not the observations. Observation have sin/cos, state only has (theta, thetadot), so if anything, the most restrictive limit would be `np.pi`

Reply:
as per your previous comment, i've removed these bound limits.